lib/src/components/NxDropdown/__tests__/NxDropdown.test.tsx

```diff
@@ -5,357 +5,301 @@
 
 describe('NxDropdown', () => {
-  let container: HTMLDivElement | null;
-
   const minimalProps = {
-        label: 'dropdown',
-        isOpen: false,
-        onToggleCollapse: () => {}
-      },
-      getShallowComponent = enzymeUtils.getShallowComponent<Props>(NxDropdown, minimalProps),
-      getMountedComponent = enzymeUtils.getMountedComponent<Props>(NxDropdown, minimalProps);
-
-  beforeEach(function() {
-    // Avoid rendering directly on the body.
-    container = document.createElement('div');
-    document.body.appendChild(container);
-  });
+    label: 'dropdown-toggle',
+    isOpen: false,
+    onToggleCollapse: () => {}
+  };
 
-  afterEach(function() {
-    if (container) {
-      document.body.removeChild(container);
-      container = null;
-    }
-  });
-
-  it('renders a button with the appropriate classes and type=button', function() {
-    const component = getMountedComponent();
-    const dropdown = component.find(AbstractDropdown);
-    const button = dropdown.find(NxButton);
-    const icon = button.find(NxFontAwesomeIcon);
-
-    expect(dropdown).toHaveClassName('.nx-dropdown');
-    expect(button).toHaveClassName('.nx-dropdown__toggle');
-    expect(button).toHaveProp('variant', 'tertiary');
-    expect(button).toHaveProp('type', 'button');
-    expect(button.childAt(0)).toContainReact(<span className="nx-dropdown__toggle-label">dropdown</span>);
-    expect(icon).toHaveProp('icon', faCaretDown);
-  });
+  const quickRender = rtlRender(NxDropdown, minimalProps);
+  const renderEl = rtlRenderElement(NxDropdown, minimalProps);
 
   it('renders the button according to the supplied variant', function() {
-    let component = getMountedComponent();
-    expect(component.find(NxButton)).toHaveProp('variant', 'tertiary');
+    const { rerender } = quickRender();
+
+    expect(screen.getByRole('button')).toHaveClass('nx-btn--tertiary');
 
-    component = getMountedComponent({ variant: 'primary' });
-    expect(component.find(NxButton)).toHaveProp('variant', 'primary');
+    rerender(<NxDropdown {...minimalProps} variant="primary" />);
+    expect(screen.getByRole('button')).toHaveClass('nx-btn--primary');
 
-    component = getMountedComponent({ variant: 'secondary' });
-    expect(component.find(NxButton)).toHaveProp('variant', 'secondary');
+    rerender(<NxDropdown {...minimalProps} variant="secondary" />);
+    expect(screen.getByRole('button')).toHaveClass('nx-btn--secondary');
 
-    component = getMountedComponent({ variant: 'error' });
-    expect(component.find(NxButton)).toHaveProp('variant', 'error');
+    rerender(<NxDropdown {...minimalProps} variant="error" />);
+    expect(screen.getByRole('button')).toHaveClass('nx-btn--error');
   });
 
   it('correctly renders the menu based on isOpen prop', function() {
-    let component = getMountedComponent({ isOpen: true });
-    expect(component.find(NxDropdownMenu)).toExist();
+    const { container, rerender } = quickRender({ ...minimalProps, isOpen: true });
```

Comment:
One of the main features of `quickRender` is that it automatically includes the `minimalProps`; you only have to specify the extras
```suggestion
    const { container, rerender } = quickRender({ isOpen: true });
```